spec/event_tracer/datadog_logger_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper'
+
+describe EventTracer::DatadogLogger do
+
+  INVALID_PAYLOADS ||= [
+    nil,
+    [],
+    Object.new,
+    'string',
+    10,
+    :invalid_payload
+  ].freeze
+
+  let(:datadog_payload) { nil }
+  let(:mock_datadog) { MockDatadog.new }
+
+  subject { EventTracer::DatadogLogger.new(mock_datadog) }
+
+  shared_examples_for 'processes_hashed_inputs' do
```

Comment:
Do we use this shared examples anywhere?

Reply:
my bad, missed this